projects/plugins/jetpack/modules/likes.php

```diff
@@ -422,49 +439,55 @@ function likes_edit_column( $column_name, $post_id ) {
 	}
 
 	/**
-	* Add a "Likes" column header to the post edit table in wp-admin.
-	*
-	* @param array $columns
-	* @return array
-	*/
-	function add_like_count_column( $columns ) {
+	 * Add a "Likes" column header to the post edit table in wp-admin.
+	 *
+	 * @param array $columns - array of columns in wp-admin.
+	 * @return array $columns - column in wp-admin.
```

Comment:
This doesn't seen quite accurate.

There's no real need to describe the return value (phpcs doesn't insist), but if you really want to I might do something like
```suggestion
	 * @return array $columns with "likes" added.
```